### DISCUSSION TITLE
Disproportionate Volume Control

### DISCUSSION THREAD

**Discussion by kzssc** [MonitorControl/MonitorControl#1056] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Control (volume + brightness) not working proportionally.  Another way to phrase the issue would be that the controls are not using up the entire scale/bar. 

Volume reaches max with a tiny portion of the volume control. (like 1/10). Current workaround is using the "fine OSD scale". 
Brightness works better but has the same problem, rather than using 1/10, it uses 1/2 of the scale, roughly. 

### Expected behavior

Controls utilising entire scales. 

### Anything else?

Thanks very much for the very handy app! It helped tremendously. 

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.4
- Mac model:M1 Pro 13 (w/ touchbar) 
- MonitorControl version: 4.1.0
- Monitor(s): Benq EU2880U
- Monitor Cable(s)/Connection(s): type-c <direct> type-c
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Hi there, you can rescale the DDC scale or change the skew of the scale under Displays. You need to enable Advanced

<img width="842" alt="Screen Shot 2022-06-02 at 16 01 58" src="https://user-images.githubusercontent.com/37590873/171647012-f8583bf3-c6a9-42bf-b2f7-b1f68e4b952d.png">
.

**Reply by kzssc** [reply]
superb - sorry for not noticing this earlier. Many thanks for the prompt reply :))
